sawtooth-java-wrapper-service/src/main/java/com/mycompany/blockchain/sawtooth/core/service/payment/PaymentValidator.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * 
+ */
+package com.mycompany.blockchain.sawtooth.core.service.payment;
+
+import com.mycompany.blockchain.sawtooth.core.service.IDataValidator;
+import com.mycompany.blockchain.sawtooth.loan.protobuf.Payment;
+
+import sawtooth.sdk.processor.exceptions.InvalidTransactionException;
+
+/**
+ * @author Sandip Nirmal
+ *
+ */
+public class PaymentValidator implements IDataValidator<Payment> {
+
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see com.mycompany.blockchain.sawtooth.core.service.IValidator#validate(java.lang. Object)
+	 */
+	@Override
+	public boolean validate(Payment data) throws InvalidTransactionException {
+		if (data.getId() == null || data.getId().isEmpty())
+			return false;
```

Comment:
Throw exception here and not a false type. the message should describe the error.

Reply:
Done